src/components/Progress/ProgressStackItem.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+import type {CnMods} from '../utils/cn';
+import {block} from '../utils/cn';
+
+import type {Stack} from './types';
+
+const b = block('progress');
+
+export interface ProgressStackItemProps {
+    item: Stack;
+}
+
+export function ProgressStackItem({item}: ProgressStackItemProps) {
+    const {value, color, className, theme, title, content, loading} = item;
+    const modifiers: CnMods = {
+        loading,
+    };
+
+    if (typeof color === 'undefined') {
+        modifiers.theme = theme || 'default';
+    }
+
+    if (Number.isFinite(value)) {
```

Comment:
Let's add check for `!isNaN(value)` like it was in class component

Hmm, maybe move this check to utils for reusing?

Reply:
The `Number.isFinite` static method determines whether the passed value is a finite number — that is, it checks that a given value is a number, and the number is neither positive Infinity, negative Infinity, nor NaN.